packages/v4/src/pages/index.js

```diff
@@ -184,11 +187,9 @@ const IndexPage = ({ location }) => {
                     />
                   </FlexItem>
                   <FlexItem>
-                    <Text component={TextVariants.h2}>Modular &amp; flexible</Text>
+                    <Text component={TextVariants.h2}>Make it flexible.</Text>
                     <Text component={TextVariants.p} className="pf-u-mb-3xl ws-mdx-p">
-                      Arrange self-contained components in any number of ways to build a variety of applications and
-                      interfaces. You can also use the CSS variable system to customize styles across a website or
-                      interface.
+                      We can’t be set in our ways. Open source thrives on variety, and our resources do too. Our components can be arranged in any number of ways, and our CSS variable system can be used for all kinds of customization.
```

Comment:
Is it stylistic to leave out the comma before "too"?
Open source thrives on variety, and our resources do, too.

Reply:
It's optional! We can leave it out since we already have some commas in there.